### ISSUE TITLE
“Event Organizers”, “Groups”, and “Additional individual attendees” list boxes don't work well with screen readers 

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/calendar#99]
Currently it is not possible to add or remove organizers, groups, and additional individual participants. The component in use is problematic, and may be causing similar issues wherever else it is used through the calendar app.

To replicate, with a screen reader running:

1. Add a new event
2. Select the “Participants” tab
3. Tab through the controls until you get to the “Organizer” list box.

**Expected result**: Screen readers announce the current label, role, and value of the list box.  For example “Organizers list, Daniel Montalvo” or “Groups list, ARIA”. Users can type one or more characters, arrow up and down to select, and then hit Enter to confirm selection.

**Obtained result:** Screen readers do not announce the label and value of that control. If users type some characters and then arrow through the results, nothing is announced.

**Comment by vivienlacourba** [comment]
@daniel-montalvo is that issue still relevant with the move to the new style?

**Comment by daniel-montalvo** [comment]
Hi @vivienlacourba 

Closing this, this is now very much improved. Related is https://github.com/w3c/w3c-website/issues/269
